hardhat.config.js

```diff
@@ -25,4 +26,7 @@ module.exports = {
       accounts: [`0x${process.env.ETH_ACCOUNT_KEY}`],
     }
   },
+  etherscan: {
+      apiKey: process.env.ETHERSCAN_KEY
```

Comment:
can you give this a default value? the main problem is we have to define this var when running the services in prod for no reason

Reply:
ya.. sure, I can also remove this field completely.. This is helpful to publish a contract on etherscan